### ISSUE TITLE
Create Gridset plugin

### ISSUE THREAD

**Issue by scottkellum** [at-import/Singularity#70]
Based on the output gridset is using the exact same syntax as we are for grids. Should easily be able to parse with a mixin like @import import-gridset(a, b); where a and be are the grids from gridset you want to import.

https://gridsetapp.com/documentation/sass/

**Comment by Snugug** [comment]
Agreed. I saw a bit of the actual output, we should figure this out.

**Comment by Snugug** [comment]
Maybe Ruby awesomeness?

**Comment by scottkellum** [comment]
Not sure that is needed, pretty much a matter of changing var names. I may just do this tonight.

**Comment by Snugug** [comment]
Awesome, then we can include it in 1.0!

**Comment by scottkellum** [comment]
Though, I want to get other 1.0 stuff out the door first.

**Comment by Snugug** [comment]
I'm going to mark this as 1.1 then, and we'll iterate on it once our 1.0 stuff is done.

**Comment by scottkellum** [comment]
I’m in contact with Nathan about it, sounds like the biggest issues here might be diplomacy, not code.

**Comment by Snugug** [comment]
Dip all the lomacy!

**Comment by scottkellum** [comment]
Not in scope for core. But as soon as gridset has a grid index I can whip this up easily!
